packages/babel-plugin-fbtee/src/bin/__tests__/translate-test.tsx

```diff
@@ -43,6 +51,7 @@ function testTranslateNewPhrases(options: Options) {
       options,
     );
     expect(result).toMatchSnapshot();
+    expect(console.error).toHaveBeenCalled();
```

Comment:
Since the translation is missing, this was logging to the terminal. It's now caught by Jest and verified that it is actually reported as missing.